hledger-lib/Hledger/Data/Types.hs

```diff
@@ -208,13 +213,22 @@ instance Show AmountStyle where
     (show asdecimalpoint)
     (show asdigitgroups)
 
```

Comment:
-- | The number of decimal places to display for an amount: 0 to 255, or NaturalPrecision meaning all available decimal digits, up to the 255 limit.

Reply:
PS do you think "precision" can be ambiguous here ? It's not the number of significant digits in the amount, but the number of decimal places (digits after the decimal mark). It might be a tedious as my use of "precision" may be pervasive, but in principle we could rename this DecimalPlaces = Places ...  or such ?

Reply:
Isn't there a typeclass that would let us still write `1` instead of `Precision 1` in tests, and when creating Amounts in ghci, etc. ?

And if we went that route, would displaying just `1` in the Show output work as well ? If not, it's probably not worth the inconsistency.

Reply:
The typeclass for doing that is `Num`. I'm hesitant to introduce an instance for which we don't want most of the functions used.

Reply:
I don't object to changing the name to DecimalPlaces.